app.js

```diff
@@ -28,10 +30,26 @@ app.use(cors());
 
 //Setting up server
 const server = app.listen(process.env.PORT || 8080, function () {
-  const port = server.address().port;
-  console.log("App now running on port", port);
+    const port = server.address().port;
+    console.log("App now running on port", port);
 });
 
+
+/**
+ * Gets all new course requests and updates them to pending status.
```

Comment:
Can u make sure this comment follows the pattern of the one of the other functions ?
You don't necessarily have to give the link bcs that will change depending on how we host the api and all but explaining the status given out would be helpful and saying that the course requests are updated as well :)